BDArmory/UI/VesselSpawnerWindow.cs

```diff
@@ -172,8 +172,9 @@ private void Start()
             {
                 Debug.LogError($"[BDArmory.VesselSpawnerWindow]: Failed to locate waypoint marker models: {e.Message}");
             }
-            if (WaygateCount >= 0) SelectedModel = Path.GetFileNameWithoutExtension(gateFiles[(int)SelectedGate]);
-            else Debug.LogWarning($"[BDArmory.VesselSpawnerWindow]: No waypoint gate models found in {Gatepath}!");
+            //if (WaygateCount >= 0) SelectedModel = Path.GetFileNameWithoutExtension(gateFiles[(int)SelectedGate]);
```

Comment:
With this commented out, `SelectedModel` is going to be a null/empty string until the Vessel Spawner Window opens with the appropriate waypoints section open, which could result in them not showing.